src/log.js

```diff
@@ -396,6 +420,7 @@ function createErrorVargs(var_args) {
  */
 export function rethrowAsync(var_args) {
   const error = createErrorVargs.apply(null, arguments);
+  recordAsyncErrorForTesting(error);
```

Comment:
@erwinmombay it should no - will let the next person who does a cut know to double check.